lib/bibliothecary/parsers/conda.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+
+module Bibliothecary
+  module Parsers
+    class Conda
+      include Bibliothecary::Analyser
+      FILE_KINDS = %w[manifest lockfile]
+
+      def self.mapping
+        {
+          match_filename("environment.yml") => {
+            parser: :parse_conda_environment,
+            kind: FILE_KINDS
+          },
+          match_filename("environment.yaml") => {
+            parser: :parse_conda_environment,
+            kind: FILE_KINDS
+          }
+        }
+      end
+
+      def self.parse_conda_environment(file_contents)
+        manifest = call_conda_parser_web(file_contents)
+
+        Hash[FILE_KINDS.collect { |kind| [kind, map_dependencies(manifest, kind, "runtime")] }]
+      end
+
+      def self.call_conda_parser_web(file_contents)
+        host = Bibliothecary.configuration.conda_parser_host
+        response = Typhoeus.post(
+          "#{host}/parse",
+          headers: {
+              ContentType: 'multipart/form-data'
+          },
+          # TODO: Can we ever get the filename from the method call from `match_filename` in the future? would be nice to be able to.
```

Comment:
Not sure I understand this TODO

Reply:
Updated it, and added a higher up TODO for more context!